src/it/resources/ported_test_log4j.properties

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Do we actually need to do this complex reconfiguration, rather than just downgrade the log levels?

Reply:
By default, IT uses the spark default log property file in org/apache/spark/log4j-defaults.properties in spark core jar file e.g. spark-core_2.12-3.0.0.jar. We couldn't change it.
In practice, I save a copy at "src/it/resources/log4j_default.properties". This config file only changes 'log4j.rootCategory'  from "INFO, console" to "WARN, console".
When we need to change the log level, we use this logging property,
After the test, we configure back to the default.